app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  validates :user_id, presence: true
```

Comment:
We already have the association with user established in line 2, we don't need anymore to validate presence of user_id here as no post will be saved unless it's created with an existing user